xbmc/events/BaseEvent.cpp

```diff
@@ -20,6 +20,9 @@
 
 #include "BaseEvent.h"
 #include "guilib/LocalizeStrings.h"
+#include "utils/StringUtils.h"
+
+unsigned int CBaseEvent::s_nextIndex = 0;
```

Comment:
Not a leftover. Look for the constructor changes below.

Sure, increasing timestamp granularity would be sufficient. But do we have a high resolution timer in Kodi? I'm not aware of any. That's why I took this shortish.